releases/net472/KnownIssues/613745 - Single-name references are removed by the SDK when targeting 4.7.2.md

```diff
@@ -0,0 +1,23 @@
+# Single-name references are removed by the SDK when targeting 4.7.2 and referencing a package with the same assembly.
```

Comment:
4.7.2 --> .NET Framework 4.7.2 
That change should be made throughout whenever there's a simple unqualified mention of 4.7.2.